packages/apollo-codegen-core/src/localfs.ts

```diff
@@ -0,0 +1,5 @@
+export const fs = require("fs");
```

Comment:
This is what's loaded during regular runtime, where `localfs` just points to regular `fs`.